src/Administration/Resources/app/administration/build/plugins.vite.ts

```diff
@@ -29,11 +30,32 @@ import { loadExtensions, findAvailablePorts } from './vite-plugins/utils';
 import type { ExtensionDefinition } from './vite-plugins/utils';
 import injectHtml from './vite-plugins/inject-html';
 
+/**
+ * This function returns the IP address of the container
+ * if the application is running in a Docker container.
+ */
+const getContainerIP = (): string|undefined => {
+    const interfaces = os.networkInterfaces();
+
+    let containerIP;
+
+    Object.values(interfaces).forEach(ifaces => {
+        ifaces?.forEach(iface => {
+            if (!iface.internal && iface.family === 'IPv4') {
+                containerIP = iface.address;
+            }
+        });
+    });
+
+    return containerIP;
+};
+
 const VITE_MODE = process.env.VITE_MODE || 'development';
 const isDev = VITE_MODE === 'development';
 
 // This env variable is provided by the symfony recipes
 const hasAdminRootEnv = !!process.env.ADMIN_ROOT;
+const host = process.env.VITE_HOST || process.env.IS_DOCKER ? getContainerIP() : 'localhost';
```

Comment:
you check if a `/.dockerenv` file exists, then you know you're running inside a container

Reply:
So I would go recursively all folder upwards until I found this `.dockerenv`? Or is it everytime in the root of the container?

Reply:
Then I will switch it to a `.dockerenv` check